### DISCUSSION TITLE
Simpson model 29 0-50uA DC ammeter

### DISCUSSION THREAD

**Discussion by doc1011** [realA10001986/Dash-Gauges#4] [Q&A]
Hi - I've fallen at the first hurdle! 

I've got a Simpson model 29 0-50mA DC ammeter  and the modification calls for cutting a wire to remove internal coil resistor, without disassembly. I can't see anything resembling what is described and have attached pictures. Apologies if being thick - what have I missed? Assume this will work with the circuit setup board? Cheers!

![IMG_3939](https://github.com/user-attachments/assets/8e24e457-11db-4ed6-9001-0e2f3a83a530)

![IMG_3940](https://github.com/user-attachments/assets/8ebef067-2c8f-44eb-a129-4e2ddbda94fb)

**Comment by realA10001986** [comment]
Hi, my bad...  by "disassembling" I actually meant "removing the meter's mechanics from its enclosure". You'd still need to open it up.

Edit: 

BUT: You have a 0-50uA (microAmp), not 0-50mA (milliAmp). Different thing. That one probably has different stuff inside, so I am afraid you'll have to remove the mechanics after all.

1) Open enclosure: Four screws on the back; Remove the face: Four tiny screws. Be careful not to damage the pointer when pulling the face out.
2) Remove all nuts from the terminals
3) Remove mechanics: Open the 5.5mm nuts on the back, and CAREFULLY pull out the meter from its enclosure. 


![openmeter](https://github.com/user-attachments/assets/f11c16f6-ffb4-4bf3-b3ad-d401eaac03ee)

**Reply by realA10001986** [reply]
I don't see why it wouldn't work with the CS board, provided you follow the instructions on how to adjust the trimpots.

**Reply by doc1011** [reply]
Ah ok - I'll open it up tomorrow see what's inside, and report back.... 

Cheers mate, appreciated as always

**Comment by doc1011** [comment]
Et voila - the inside - seems pretty basic! Can it be used?

![IMG_3947](https://github.com/user-attachments/assets/71d2a285-4fd8-457a-ac8d-9c36dfff3c39)

![IMG_3948](https://github.com/user-attachments/assets/4167622d-3404-4a33-97b5-cbaed1e74f8e)

![IMG_3949](https://github.com/user-attachments/assets/077a3f4a-c3b9-4e24-b704-ce6054d2bbd3)

**Comment by realA10001986** [comment]
Is there something (resistors or the likes) between the terminal screws and the meter? The two black wires coming from the meter must be directly connected to the input terminals. If there is anything inbetween, it needs to be removed. 

For my reference list, I'd like to know what there is, preferably with a pic ;) Thanks!

**Comment by doc1011** [comment]
I can't see any resistors or any electronics at all - do you think it will work? I've got a few pics below for you but happy to take more if you like!

![IMG_3958](https://github.com/user-attachments/assets/29992566-b631-4b87-ae29-81991e9bb36b)
![IMG_3957](https://github.com/user-attachments/assets/744220bc-0ccc-4889-9855-1d85ea621c3f)
![IMG_3955](https://github.com/user-attachments/assets/b0bcc745-f863-4553-8a73-80d7c335d432)
![IMG_3956](https://github.com/user-attachments/assets/4b76cc1b-cc9a-436c-9ec6-81bd4626affe)
![IMG_3953](https://github.com/user-attachments/assets/4528bf7e-a974-4f05-a966-175840bea273)

**Comment by realA10001986** [comment]
It most definitely will work!

(Odd. That's the second ammeter I came accross with nothing inside.)

**Comment by doc1011** [comment]
Phew :) Will it need anything special for R5/R6 m the board?

**Comment by realA10001986** [comment]
I thought you had a CS board? Those have trimpots pre-installed. Please see hardware page for adjustment.

Otherwise:

1) Adjust the zero point (using the screw in the center of the meter's front) to the green 0, which is about 3 steps off the zero point of your current face. 
2) Set the "full" percentage of the Roentgens to 100 in the Config Portal
3) Set the Gauge Type to "Generic analog 0-5V" in the Config Portal
4) Put a wire bridge in R6 (do not solder, just bend it so it makes contact)

Power up.

Then put an 8k2 resistor in R5 and watch the needle. If it jumps to the end of the scale, remove the resistor immediately and retry with a higher value (10k, ...), otherwise work your way down, until the needle is touching the end point softly. You can then fine-tune with a small resistor (120, 270, 470, ..) in R6 (instead of the wire).

If you need very high resistors (20k and above), change the Gauge Type to "Generic Analog 0-2.048V" and repeat.

**Comment by doc1011** [comment]
Fab, thanks - the board's on its way and looking forward to getting it all up and running to add to the rest of the kit. Your software works amazingly - I've got the whole thing integrated into HomeKit now and my kids love it :)

**Comment by realA10001986** [comment]
OK, if you need assistance don't hesitate to ask.

So there is somebody actually using MQTT? Fascinating... :)

**Comment by doc1011** [comment]
Ha, yes, I've done a deep dive in with MQTT. Have even got the TCD responding to Alexa commands, with full on routines. Loads of fun and rock solid.

**Reply by realA10001986** [reply]
Happy to hear! Apart from sending in commands with mosquitto for testing I have never really used it myself. I am more the "a light switch is a light switch" kind of guy ;)

**Comment by realA10001986** [comment]
Hi, how's your build coming along?

**Reply by doc1011** [reply]
HI - the board has arrived and is working great! I'm wondering if there might be a bug in the code as the gauge type isn't updating when saving in the settings, but will log it with the detail. I've compensated with the variable resistor and it's working great. I need to wire up the empty light (may need to print a bezel adapter to make space) and sort out the labels etc on the other gauges. Then finish off the enclosure. Good times :)

**Comment by doc1011** [comment]
Hiya mate - I'm working on the enclosure while I wait for the board - should be with me in the next few days. I've made up an STL in 3 separate pieces (I've got a small printer). Once I've confirmed the board fits and works I'll prime/paint it nice and shiny. 

![IMG_4213](https://github.com/user-attachments/assets/39a2a59e-b955-471e-9192-6dc9facb60a3)
